app/src/main/res/layout/fragment_fingerprint_dialog.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  -->
+
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                             xmlns:app="http://schemas.android.com/apk/res-auto"
+                                             xmlns:tools="http://schemas.android.com/tools"
+                                             android:layout_width="match_parent"
+                                             android:layout_height="match_parent">
+
+    <TextView
+            android:text="@string/fingerprint_dialog_title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/title"
+            android:fontFamily="sans-serif"
+            android:textStyle="normal"
+            android:textColor="@color/textGrey"
+            android:lineSpacingExtra="8.5sp"
+            app:layout_constraintTop_toTopOf="parent" android:layout_marginTop="44dp" android:textSize="17sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" android:layout_marginStart="52dp"
+            android:layout_marginEnd="51dp"/>
```

Comment:
Nit: The layout of these looks somewhat haphazard. Recommend we adopt the Android style in Android Studio.